lib/html_parser.dart

```diff
@@ -733,23 +733,20 @@ class HtmlRichTextParser extends StatelessWidget {
                               ? double.parse(node.attributes['height'])
                               : null),
                       scale: imageProperties?.scale ?? 1.0,
-                      matchTextDirection:
```

Comment:
It looks like there are quite a few code formatting changes in this pull request. Do you mind running `flutter format` on this file to keep the formatting consistent?

Reply:
It seems that Android Studio keeps formatting the code whenever I use Ctrl+S.